backend/src/main/java/quarano/department/TrackedCaseDataInitializer.java

```diff
@@ -123,7 +123,7 @@ public void initialize() {
 		LocalDate startG = LocalDate.now().minusWeeks(1).plusDays(2);
 		LocalDate endG = start.plusWeeks(2);
 
-		cases.save(new TrackedCase(TRACKED_CASE_GUSTAV, person4, CaseType.INDEX, department1, null)
+		cases.save(new TrackedCase(TRACKED_CASE_GUSTAV, person4, CaseType.CONTACT, department1, null)
```

Comment:
Why did you make this change?

Reply:
Because of the existing tests. There are no more "first symptoms" for index cases and gustav was an index case. Subsequently, asserts around this guy started failing because they expected this kind of data. And so his type changed to contact, thus in turn "recovered" his tests without changes.

Reply:
I cannot say whether the test cases of our testers will then no longer work.
In any case, the data are not consistent. With an infection, he's an index case.